libmamba/src/core/history.cpp

```diff
@@ -14,7 +14,7 @@ namespace mamba
 {
     History::History(const fs::path& prefix)
         : m_prefix(prefix)
-        , m_history_file_path(m_prefix / "conda-meta" / "history")
+        , m_history_file_path(fs::absolute(m_prefix / "conda-meta" / "history"))
```

Comment:
the `fs::absolute` shouldn't really change anything, right?

Reply:
Sorry for the delay.
Yes this doesn't change the behavior of the code, however it makes the log in the constructor clearer as to which location the file is. The relative path was misleading me when I was debugging and felt it would help later maintenance to have it absolute so that any logging will always be precise.